lms/djangoapps/teams/static/teams/templates/team-profile.underscore

```diff
@@ -4,8 +4,10 @@
                 data-read-only="<%- readOnly %>"
                 data-user-create-comment="<%- !readOnly %>"
                 data-user-create-subcomment="<%- !readOnly %>">
-            <% if (!readOnly) { %>
-                <button type="button" class="btn new-post-btn"><span class="icon fa fa-edit new-post-icon" aria-hidden="true"></span><%- gettext("New Post") %></button>
+            <% if ( !readOnly) { %>
+                <button type="button" class="btn-neutral btn-small new-post-btn">
+                    <i class="icon fa fa-edit" aria-hidden="true"></i> <%- gettext("New Post") %>
```

Comment:
Thanks for catching this, @dianakhuang. Yes, no more i tags! @ssemenova's PR merged yesterday (#12579), so likely this is a bad rebase.